jmeter-service/chart/values.schema.json

```diff
@@ -39,9 +38,6 @@
               },
               "apiValidateTls": {
                 "type": "boolean"
-              },
-              "token": {
-                "pattern": "^[A-Za-z0-9-.]{2,63}$"
```

Comment:
It would be great if we could keep that constraint in place. I assume you removed it because the token has the option to not be defined now, right?
I think we can fix that by introducing a [oneOf](https://json-schema.org/understanding-json-schema/reference/combining.html#oneof) constraint in the JSON Schema here and also for the `helm-service`.

Reply:
Yeah exactly this and I couldn't figure out the syntax to make it conditional. I'll have another crack at this using oneOf